Source/Dafny/Rewriter.cs

```diff
@@ -118,6 +119,36 @@ public abstract class IRewriter {
       Contract.Requires(program != null);
     }
   }
+  // intern project 
+
+  class tobiRewriter : IRewriter {
```

Comment:
Could you add a comment above this class which contains a code examples of when this class shows an error?